BUILD.bazel

```diff
@@ -4,12 +4,47 @@
 
 load("@io_kubernetes_build//defs:run_in_workspace.bzl", "workspace_binary")
 load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
+load("@io_bazel_rules_go//go:def.bzl", "nogo")
 
 workspace_binary(
     name = "go",
     cmd = "@go_sdk//:bin/go",
 )
 
+nogo(
+    name = "nogo_vet",
+    config = "nogo_config.json",
+    visibility = ["//visibility:public"],
+    # These deps enable the analyses equivalent to running "go vet".
```

Comment:
Where do I go to find new ones of these to add?

Reply:
`go doc cmd/vet` was what I used to figure out what `go vet` actually runs (and which ones it doesn't run). I'll add a comment.

We can also write our own using the same framework and add them to this, if we have reason to do so in the future.